charts/scorpio-broker-dist/values.yaml

```diff
@@ -0,0 +1,702 @@
+## namespace for deployment
+namespace: scorpio-broker
+################################### Eureka Server #######################################################################
+eureka:
+## Enable the eureka deployment & service
+  enabled: true 
+  name: eureka
+## Number of eureka instances to deploy
+##
+  replicas: 1
+## Scorpio Broker eureka image version
+## ref: https://hub.docker.com/r/scorpiobroker/scorpio
+## Please use only latest version of docker images as few older images might not work with kubernetes. 
+##
+  image:
+   repository: scorpiobroker/scorpio
```

Comment:
According to the Chart.yaml, this deploys version 1.16.0 as a default. All image tags are "latest". This will lead to unexpected versions being deployed. Please use fixed versions as a default, to avoid the components being silently updated.

Reply:
We have incorporated the review comment. Please review again